GoogleForCreators/web-stories-wp
# Editor: Add support for `amp-story-audio-sticker`

## Summary


This PR adds support for [amp-story-audio-sticker](https://amp.dev/documentation/components/amp-story-audio-sticker).

## User-facing changes



https://github.com/GoogleForCreators/web-stories-wp/assets/75766877/d422dc47-b1b7-4d89-b72e-9fa65326f532

https://github.com/GoogleForCreators/web-stories-wp/assets/75766877/86945bcb-6f0b-4bf1-8d94-f18244dca65b

## Testing Instructions






This PR can be tested by following these steps:

1. Press the Insert Audio Sticker quick action button. Please note that this button is only visible if the story has video with audio/background audio.
2. The `headphone-cat` variant of the sticker is added by default. To change this and other options, select the sticker element and view the style panel.


## Reviews

### Does this PR have a security-related impact?
Yes.


### Does this PR change what data or activity we track or use?
No.


### Does this PR have a legal-related impact?
No.


## Checklist




---



Fixes #13440
